airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -24,6 +24,46 @@
 Changelog
 ---------
 
+7.2.0
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+
+Features
+~~~~~~~~
+
+* ``Add option to wait for completion on the EmrCreateJobFlowOperator (#28827)``
+* ``Add transfer operator S3 to (generic) SQL (#29085)``
+* ``add retries to stop_pipeline on conflict (#29077)``
+* ``Add log for AWS Glue Job Console URL (#28925)``
+* ``Enable individual trigger logging (#27758)``
```

Comment:
@dstandish  this entry is in several provider releases. This feature is suppose to be released in airflow 2.6.0 however in providers we don't cherry pick we release everything that is merged. Is that OK for this case?

Reply:
Hi @eladkal 

> however in providers we don't cherry pick we release everything that is merged. Is that OK for this case?

Can you clarify what you mean?  I.e. what's the "that" that you are wondering if it is ok?

Reply:
Oh you are saying, are these changes compatible with airflow < 2.6 -- yes, that is the intention.  And I can help test this in the coming week.

Reply:
Cool! Thanks!

Reply:
OK i built the provider packages from main which have task log handlers, and i tested the handler functionality with 2.5.1 and did not observe any issues.

Reply:
Good call @eladkal ! Thanks for checking Daniel. We have so far very rudimentary "import" checks for back-compat, but this one has indeed higher probablity of breaking compat.